### ISSUE TITLE
Can you describe the process of data processing in detail?

### ISSUE THREAD

**Issue by xuyaojian123** [husnejahan/DeepAR-pytorch#3]
In the preprocess_elect.py file, how do you process the data, and how are the labels of the data divided? Thank you very much

**Comment by husnejahan** [comment]
First you need to understand how a single time series works, here 370 time series together. Also if you could understand the following parameters:

  window_size = 192
  stride_size = 24
  num_covariates = 4

pre-processing would be clear then.

**Comment by xuyaojian123** [comment]
> 

Thanks, I seem to understand, are these hyperparameters experimentally derived?
window_size means that each label consists of 192 consecutive data
stride_size means to skip 24 steps to take the next training data
num_covariates means that each data has four covariates to help predict

**Comment by husnejahan** [comment]
Yes, You can perform you own tuning.
